### PR TITLE
Translate README to Chinese.

### PR DESCRIPTION
**Pull request by dingjs**
Translate README to Chinese.

### CODE REVIEW

**Comment by deekoder** [comment]
waiting for @yihuaf here

**Comment by dingjs** [comment]
ping @yihuaf 
Thanks for you effort,These are the last PRs of the translation.

File: README_zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+
+   
+ * 步骤5 - 你应该可以看见gradle进行build并创建了一个工程，现在我们可以开始敲代码了。
+
+![minio_ANDROID6](https://github.com/minio/android-photo-app/blob/master/docs/screenshots/minio-ANDROID6.jpg?raw=true)
+
+
+## 3. App Layout
+
+删除默认的Hello World TextView。
```
Comment (yihuaf) [inline README_zh_CN.md]: 默认 -> 初始

File: README_zh_CN.md [hunk 2]
```diff
@@ -0,0 +1,310 @@
+# Android Photo App [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io) 
+
+![minio_ANDROID1](https://github.com/minio/android-photo-app/blob/master/docs/screenshots/minio-ANDROID1.jpg?raw=true)
+
+本示例将会指导你如何构建一个简单的Android Photo app。在这个app中，你将会学习一个Android client是如何访问Photo API Service并随机加载一张图片。你可以通过[这里](https://github.com/minio/android-photo-app)获取完整的代码，代码是以Apache 2.0 License发布的。
+
+
+## 1. 依赖
+
+我们将使用Android Studio进行开发。这个app也会访问我们发布的Photo API Service来随机获获取一张图片的presigned url。
```
Comment (yihuaf) [inline README_zh_CN.md]: 获获取 -> 获取

**Review by yihuaf — CHANGES_REQUESTED** [review]

**Review by yihuaf — APPROVED** [review]

**Pull request merged**